### ISSUE TITLE
Argument -Q not working (check_arg_types error)

### ISSUE THREAD

**Issue by HeatherKates** [juliema/aTRAM#243]
Hello, 

As of my first attempt to use the updated version of atram on hipergator, option -Q is no longer working. I get the following error message:
`atram.py -b P023_WA01 --cpus 1 -Q /ufrc/soltis/gwstull/Rosales/Reference_sequences/phylogenetic_loci_arabidopsis_proteins -i 5 --evalue 0.0002 -a spades --timeout 1000 -o Assemblies_Phylo/P023_WA01.Locus.atram.fasta --log-file P023_WA01.Locus.log`

> Traceback (most recent call last):
> 
>   File "/opt/2.1/atram.py", line 216, in <module>
> 
>     assemble(ARGS)
> 
>   File "/opt/2.1/lib/core_atram.py", line 17, in assemble
> 
>     queries = split_queries(args)
> 
>   File "/opt/2.1/lib/core_atram.py", line 111, in split_queries
> 
>     path = join(args['temp_dir'], 'queries')
> 
>   File "/usr/lib/python3.5/posixpath.py", line 89, in join
> 
>     genericpath._check_arg_types('join', a, *p)
> 
>   File "/usr/lib/python3.5/genericpath.py", line 143, in _check_arg_types
> 
>     (funcname, s.__class__.__name__)) from None
> 
> TypeError: join() argument must be str or bytes, not 'NoneType'

If I run the exact same command but with "-q", atram runs without error. I have not changed my reference file since running the old version of atram successfully. I attached the reference (-Q) file. I changed it to a .txt file so I could upload to atram, it is named .fasta on hipergator.
[phylogenetic_loci_arabidopsis_proteins.txt](https://github.com/juliema/aTRAM/files/2988997/phylogenetic_loci_arabidopsis_proteins.txt)

**Comment by HeatherKates** [comment]
Update: I mentioned this to Ryan and he said he had this error too, and that when he added the -t argument back in it resolved it. I tried that, and it worked for me too. I thought that the update eliminated the need for the -t argument. I will put it back in and proceed, let me know if you have other thoughts. Thank you!

**Comment by juliema** [comment]
Hey thanks for letting us know about this. Check the runs with the -q flag that it is assembling each gene independently. It might blast everything in that file as one run.

On Wed, Mar 20, 2019 at 8:55 AM Heather Kates <notifications@github.com<mailto:notifications@github.com>> wrote:

Update: I mentioned this to Ryan and he said he had this error too, and that when he added the -t argument back in it resolved it. I tried that, and it worked for me too. I thought that the update eliminated the need for the -t argument. I will put it back in and proceed, let me know if you have other thoughts. Thank you!

**Comment by rafelafrance** [comment]
Super annoying to debug because it's working locally. This may take a bit to fix.

**Comment by juliema** [comment]
ok no prob.

On Fri, Apr 19, 2019 at 7:04 AM rafe <notifications@github.com<mailto:notifications@github.com>> wrote:

Super annoying to debug because it's working locally. This may take a bit to fix.

**Comment by rafelafrance** [comment]
I tested this on a system with python 3.5 (Moose) and it appears to work now. Commit 00e67ed

**Comment by HeatherKates** [comment]
Hi Rafe and Julie, sorry it took me a month to test this, but of course I'm only working on hipergator so I just want to confirm that I need to request that hipergator updates atram/2.1 with the most recent commit before I test? (I did test with what's on there now as 2.1 and it gets the same error, so I'm assuming that's the case but double checking before I ask them to update)

**Comment by juliema** [comment]
Hey heather yes we need to update the version on hipergator.

We just fixed a few other tiny bugs so will you ask them to upload version 2.1.2?

thanks!

On Wed, May 29, 2019 at 12:37 PM Heather Kates <notifications@github.com<mailto:notifications@github.com>> wrote:

Hi Rafe and Julie, sorry it took me a month to test this, but of course I'm only working on hipergator so I just want to confirm that I need to request that hipergator updates atram/2.1 with the most recent commit before I test? (I did test with what's on there now as 2.1 and it gets the same error, so I'm assuming that's the case but double checking before I ask them to update)
